content/docs/start/experiments.md

```diff
@@ -170,14 +170,81 @@ $ git add dvc.lock params.yaml prc.json roc.json scores.json
 $ git commit -a -m "Preserve best random forest experiment"
 ```
 
-> `dvc push` only uploads persistent experiments that have been committed to
-> Git. The other experiments will not be pushed to the remote. See
-> `dvc exp push` and `dvc exp pull` for how to share other experiments.
+## Sharing experiments
+
+After committing the best experiment to our Git branch, we can
+[store and share](/doc/start/data-versioning#storing-and-sharing) it remotely
+like any other iteration of the pipeline.
+
+```dvc
+dvc push
+git push
+```
+
+<details>
+
+### 💡 Expand to try out sharing with simple remotes.
+
+A DVC remote stores the experiment data, and a Git remote stores the code,
+parameters, and other metadata associated with the experiment. DVC supports
+various types of remote storage (local file system, SSH, Amazon S3, Google Cloud
```

Comment:
Not sure we need all the Git remote related info? Makes the section quite long too. It's hidden so not a huge deal but it's typically better to avoid explaining other tools, even Git. Even explaining DVC remotes much in this doc is probably unnecessary (already introed in https://dvc.org/doc/start/data-versioning#storing-and-sharing).

Reply:
My rec is to focus on the application of DVC remotes for the `exp` context.

Reply:
That said I may be missing something (didn't read this through in detail) — e.g. that `exp push/pull` use Git remotes. But if it's possible to reduce X remote explanations at least, that would be best I think.

Reply:
Yeah, this whole section was a bit of a challenge because:
1) It requires both a Git remote and DVC remote to actually try it out, so I wasn't sure whether to review those concepts so users who come straight to this section can try it, or to ignore them and keep it focused on `exp push/pull` commands only.
2) Without a more complex example of a shared project, which seems out of scope here, there's not much to show the utility of `exp push/pull`.

Reply:
I shortened it, but maybe it's still too much? Let me know your thoughts.

Reply:
Here's the updated details section to save you some time:

```
### 💡 Expand for more info on remote storage.

The commands in this section require both a default DVC remote and a Git remote.
A DVC remote stores the experiment data, and a Git remote stores the code,
parameters, and other metadata associated with the experiment. DVC supports
various types of remote storage (local file system, SSH, Amazon S3, Google Cloud
Storage, HTTP, HDFS, etc.). The Git remote is often a central Git server
(GitHub, GitLab, BitBucket, etc.).
```

Reply:
👋 @jorgeorpinel, could you review this one edit before I merge?

Reply:
@jorgeorpinel Just pinging you again on this one 😄 . Let me know if you have specific concerns or even think this section is unhelpful/doesn't belong in getting started. I'm conflicted about it myself.

Reply:
Oops @dberenbaum sorry this fell through the cracks, checking now...

Reply:
Yeah def. very nicely summarized. I'd just add some links perhaps.